autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/LoadAlertRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.periscope.api.model;
```

Comment:
Similarly here. The only difference between the Request and Response is the 'id' in the response. Base class + inherit?